mulighetsrommet-api/src/main/resources/db/migration/V16__add_table_ansatt_tiltaksgjennomforinger.sql

```diff
@@ -0,0 +1,13 @@
+create table ansatt_tiltaksgjennomforing
+(
+    id                      uuid      default gen_random_uuid(),
```

Comment:
Denne kolonnen virker overflødig, kan den fjernes?

Reply:
Ja, den brukes ikke til noe. Jeg tenkte det var ryddig å ha en eksplisitt id for radene i tabellen uansett, men det er kanskje bare overflødig når jeg allikevel sier at primary-key er navident + tiltaksgjennomføringen sin egen id

Reply:
Jeg fjerner id'en